.golangci.yml

```diff
@@ -0,0 +1,64 @@
+run:
+  timeout: 3m
+  tests: false
+
+linters:
+  enable:
```

Comment:
that's quite exhaustive list. Let's leave it for now as it is as long as we agree to reduce it in the following PR adapting the code according to the linter. How does it sound?